packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -103,6 +106,21 @@ export default abstract class EasCommand extends Command {
    */
   static contextDefinition: ContextInput = {};
 
+  /**
+   * The user session manager. Responsible for coordinating all user session related state.
+   * If needed in a subclass, use the SessionManager ContextOption.
+   */
+  private sessionManagerInternal?: SessionManager;
+
+  /**
+   * The analytics manager. Used for logging analytics.
+   * It is set up here to ensure a consistent setup.
+   */
+  private analyticsInternal?: AnalyticsWithOrchestration;
+
+  protected baseErrorMessage: string = '';
```

Comment:
Why is it an empty string? Maybe make it optional if it makes sense it's optional:
```suggestion
  protected baseErrorMessage?: string;
```